src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -191,6 +191,16 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
             @Config.LangKey("config." + MOD_ID + ".general.fallable.propagateCollapseChance")
             public double propagateCollapseChance = 0.55;
 
+            @Config.Comment("Horizontal radius of the support range of support beams.")
+            @Config.RangeInt(min = 0, max = 8)
+            @Config.LangKey("config." + MOD_ID + ".general.fallable.supportBeamRangeHor")
```

Comment:
Missing the appropriate lang keys (see the config section in `en_us.lang`) for these two config options.